### PR TITLE
Update q14 vi faq.md (Fix #1665)

### LINKED ISSUE #1665: Fix Links and images in Q14
> 1. Go to this **atlas box page.**

This link give 404 error. It should be changed to: https://app.vagrantup.com/ole/boxes/jessie64
The image should able be updated accordingly.

>You should first check if **Hashicorp's atlas** is up and running by looking at the left sidebar:

The above link also gives 404 Error.A more accurate way to check the status would be here: https://status.hashicorp.com/

### PR DESCRIPTION
**Pull request by sahilph**
Fixes Issue #1665 
https://rawgit.com/sahilph/sahilph.github.io/update-q14-vi-faq.md/#!pages/vi/vi-faq.md

### CODE REVIEW

**Comment by ArashDai** [comment]
Could you add a rawgit link

**Comment by sahilph** [comment]
rawgit link added

**Comment by Jackbui96** [comment]
Can you remove/change this line since the line does not match with what we are doing.
Line: You should first check if Hashicorp's atlas is up and running by looking at the left sidebar:

We are not checking for anything at the left sidebar correct?

**Comment by sahilph** [comment]
Oh yes. You are right. I'll update it.

File: pages/vi/vi-faq.md [hunk 1]
```diff
@@ -83,13 +83,13 @@
 
 #### Q14: When I first run BeLL with the "vagrant up" command, why does the download fail?
 
-+ You should first check if [Hashicorp's atlas](https://atlas.hashicorp.com/boxes/search) is up and running by looking at the left sidebar:
++ You should first check if [Hashicorp's atlas](https://status.hashicorp.com) is up and running by looking at the left sidebar:
 
   ![Left Sidebar](images/vi-atlas-status.png)
```
Comment (Jackbui96) [inline pages/vi/vi-faq.md]: Can you also change"[Left Sidebar]" to something like "[Atlas Status]".

Comment (sahilph) [reply pages/vi/vi-faq.md]: I have made some changes. please check

**Comment by sahilph** [comment]
I have made some changes. please check

File: pages/vi/vi-faq.md [hunk 2]
```diff
@@ -83,13 +83,13 @@
...
 
-+ You should first check if [Hashicorp's atlas](https://atlas.hashicorp.com/boxes/search) is up and running by looking at the left sidebar:
++ You should first check if [Hashicorp's atlas](https://status.hashicorp.com) is up and running by looking at the status bar just below the logo:
 
   ![Left Sidebar](images/vi-atlas-status.png)
 
   If Atlas is operational, then maybe your download is being interrupted, you can try to download and set up the big Vagrant box file manually:
 
-  1. Go to this [atlas box page](https://atlas.hashicorp.com/ole/boxes/jessie64/).
+  1. Go to this [atlas box page](https://app.vagrantup.com/ole/boxes/jessie64).
```
Comment (xyb994) [inline pages/vi/vi-faq.md]: This would be `Vagrant box page` here?

File: pages/vi/vi-faq.md [hunk 3]
```diff
@@ -83,13 +83,13 @@
 
 #### Q14: When I first run BeLL with the "vagrant up" command, why does the download fail?
 
-+ You should first check if [Hashicorp's atlas](https://atlas.hashicorp.com/boxes/search) is up and running by looking at the left sidebar:
++ You should first check if [Hashicorp's atlas](https://status.hashicorp.com) is up and running by looking at the status bar just below the logo:
```
Comment (xyb994) [inline pages/vi/vi-faq.md]: It's still saying "Hashicorp's atlas" here. To be precise, we should be looking at "Vagrant Cloud" status that handles "Vagrant box publishing, hosting, and management", right?

A closer screenshot only showing "Vagrant Cloud" status would also help.

File: pages/vi/vi-faq.md (hunk 2 above)
Comment (sahilph) [reply pages/vi/vi-faq.md]: Yes. Even more accurately should it be `Vagrant Cloud Box Page` ?

**Comment by sahilph** [comment]
@xyb994 Shall I crop the `How to use this box with Vagrant` part ?

**Comment by sahilph** [comment]
Also please check the status image. Shall I crop it too ?

**Comment by xyb994** [comment]
@sahilph Sure, cropping both images to the highlighted red box would help reduce the size.

**Comment by kylemathias** [comment]
@sahilph you can crop that one as well. Let us know in gitter when you do.

**Comment by sahilph** [comment]
@kylemathias @xyb994 Please check the cropped images.

**Pull request merged**